packages/ui/src/drawer/DrawerActions.tsx

```diff
@@ -41,6 +41,7 @@ const useStyles = makeStyles(
 
 export type DrawerActionsProps = Omit<HTMLAttributes<HTMLDivElement>, 'color'>;
 
+/** @deprecated - Please use `<Drawer>` component */
```

Comment:
I think we should not deprecate usage of DrawerActions. It should be composable so we can use it anywhere